.github/workflows/tests.yml

```diff
@@ -178,13 +180,25 @@ jobs:
     - name: Install Ghidrathon
       run : |
         mkdir ./.github/ghidrathon
+        .github/ghidra/ghidra_10.3_PUBLIC/ghidraRun
+        mkdir ~/.ghidra/.ghidra_10.3_PUBLIC/Extensions
         wget "https://github.com/mandiant/Ghidrathon/archive/refs/tags/v2.1.0.zip" -O ./.github/ghidrathon/ghidrathon-2.1.0.zip
         unzip .github/ghidrathon/ghidrathon-2.1.0.zip -d .github/ghidrathon/
         workdir=$(pwd)
         /opt/gradle/gradle-7.3/bin/gradle -p ./.github/ghidrathon/Ghidrathon-2.1.0/ -PGHIDRA_INSTALL_DIR=$workdir/.github/ghidra/ghidra_10.3_PUBLIC 
-        unzip .github/ghidrathon/Ghidrathon-2.1.0/dist/*.zip -d $workdir/.github/ghidra/ghidra_10.3_PUBLIC/Extensions
+        unzip .github/ghidrathon/Ghidrathon-2.1.0/dist/*.zip -d ~/.ghidra/.ghidra_10.3_PUBLIC/Extensions
     - name: Install pyyaml
       run: sudo apt-get install -y libyaml-dev
     - name: Install capa
-      run: pip install -e .[dev]
+      run: | 
+        pip install -e .[dev]
+        git submodule update --init rules
```

Comment:
rules and testfiles should already be checked out, due to here: https://github.com/mandiant/capa/blob/e1d4fd44f5cc1e0ccbbde8d92089ea1d5e9cf382/.github/workflows/tests.yml#L152
right?

Reply:
I was under the same impression and have updated the config to get rid of this line (See a7a386759395b9162deaa3d63816a0a5da452a4c). I think it may have been an issue as I was testing using a local runner from another open-source project (https://github.com/nektos/act) that may not have supported the action fully.

Reply:
ah ok that makes sense. i haven't used a local Action runner but it seems like a great idea. let me know if you'd recommend this one.

Reply:
Act worked very well on my end! My issues stemmed from not configuring my github authentication stuff for it to use.